translations/ja.json

```diff
@@ -761,55 +761,55 @@
+	"Unpausing node {{.name}} ...": "{{.name}} ノードを再稼働させています ...",
+	"Unpausing node {{.name}} ... ": "{{.name}} ノードを再稼働させています ... ",
+	"Unset the KUBECONFIG environment variable, or verify that it does not point to an empty or otherwise invalid path": "環境変数 KUBECONFIG をセット解除するか、同変数が空または不正なパスに設定されていないことを確認してください",
+	"Unset variables instead of setting them": "変数をセットする代わりに、セット解除してください",
+	"Update Docker to the latest minor version, this version is unsupported": "Docker を最新のマイナーバージョンに更新してください (このバージョンは未サポートです)",
+	"Update kubeconfig in case of an IP or port change": "IP アドレスやポート番号が変わった場合に kubeconfig を更新してください",
+	"Update server returned an empty list": "空リストを返したサーバーを更新してください",
+	"Updating the running {{.driver_name}} \"{{.cluster}}\" {{.machine_type}} ...": "実行中の {{.driver_name}} 「{{.cluster}}」 {{.machine_type}} を更新しています...",
+	"Upgrade to QEMU v3.1.0+, run 'virt-host-validate', or ensure that you are not running in a nested VM environment.": "QEMU v3.1.0 以降にアップグレードするか、'virt-host-validate' を実行するか、ネストされた VM 環境中で実行されていない事を確認してください。",
 	"Upgrading from Kubernetes {{.old}} to {{.new}}": "Kubernetes を {{.old}} から {{.new}} にアップグレードしています",
-	"Usage": "",
-	"Usage: minikube completion SHELL": "",
-	"Usage: minikube delete": "",
-	"Usage: minikube delete --all --purge": "",
-	"Usage: minikube node [add|start|stop|delete|list]": "",
-	"Usage: minikube node delete [name]": "",
-	"Usage: minikube node list": "",
-	"Usage: minikube node start [name]": "",
-	"Usage: minikube node stop [name]": "",
-	"Use \"{{.CommandPath}} [command] --help\" for more information about a command.": "",
-	"Use 'kubect get po -A' to find the correct and namespace name": "",
-	"Use -A to specify all namespaces": "",
-	"Use SSH connection instead of HTTPS (port 2376)": "",
-	"Use SSH for running kubernetes client on the node": "",
-	"Use VirtualBox to remove the conflicting VM and/or network interfaces": "",
-	"Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'.": "",
+	"Usage": "使用法",
+	"Usage: minikube completion SHELL": "使用法: minikube completion SHELL",
+	"Usage: minikube delete": "使用法: minikube delete",
+	"Usage: minikube delete --all --purge": "使用法: minikube delete --all --purge",
+	"Usage: minikube node [add|start|stop|delete|list]": "使用法: minikube node [add|start|stop|delete|list]",
+	"Usage: minikube node delete [name]": "使用法: minikube node delete [ノード名]",
+	"Usage: minikube node list": "使用法: minikube node list",
+	"Usage: minikube node start [name]": "使用法: minikube node start [ノード名]",
+	"Usage: minikube node stop [name]": "使用法: minikube node stop [ノード名]",
+	"Use \"{{.CommandPath}} [command] --help\" for more information about a command.": "コマンドに関する追加情報は「{{.CommandPath}} [command] --help」を使用してください。",
+	"Use 'kubect get po -A' to find the correct and namespace name": "'kubect get po -A' を使用して、妥当なネームスペース名を見つけてください",
```

Comment:
Just comment.
May it be typo in original text? :thinking:

Reply:
@atoato88 thanks for your comment. It looks so. I'll make an issue to fix typos including it.